integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -1146,8 +1144,7 @@ fn test_resharding_v3_yield_resume() {
             vec![account_in_left_child, account_in_right_child],
             ReceiptKind::PromiseYield,
         ))
-        // TODO(resharding): test should work without changes to num_rpcs and track_all_shards
-        .num_rpcs(0)
+        // TODO(resharding): test should work without changes to track_all_shards
         .track_all_shards(true)
```

Comment:
We still have the issue of `track_all_shards` must be `true`, right?

Reply:
yes